README.md

```diff
@@ -64,7 +64,10 @@ Afterwards, you should run the following commands
 $ cd server
 $ python manage.py syncdb
 ```
-During syncdb, make sure that you create a user as we need it later on.
+Manually create the fuzzmanager user
```

Comment:
I don't remember creating this user manually but maybe I logged in with the user to create it. Anyway, the correct way to fix this is to use get_or_create for the user in get_auth_token.py. Something like

`(user, _) = User.objects.get_or_create(username=label)`

in server/crashmanager/management/commands/get_auth_token.py should fix the issue. I can fix this, or you can update your PR, as you want :)